### PR TITLE
mgba: git-20160325 -> 0.5.1

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

Update to 0.5.1
###### Things done
- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/misc/emulators/mgba/default.nix [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-  version = "20160325";
+  name = "mgba-${version}";
+  version = "0.5.1";
 
-  src = fetchgit {
-    url = "https://github.com/mgba-emu/mgba.git";
-    rev = "be2641c77b4a438e0db487bc82b43bc27a26e0c2";
-    sha256 = "1wxywfbkgqvb0j9cyz4nwsfzhxrdjcmvz1k7rljmy4bz1pjcglj1";
+  src = fetchurl {
+    url = "https://github.com/mgba-emu/mgba/archive/${version}.tar.gz";
```
Comment (Mic92) [inline pkgs/misc/emulators/mgba/default.nix]: how about `fetchFromGitHub` instead of fetchurl?

Comment (AndersonTorres) [inline pkgs/misc/emulators/mgba/default.nix]: I don't know about it... There is some sensible difference?

Comment (globin) [inline pkgs/misc/emulators/mgba/default.nix]: GitHub periodically regenerates the archives, changing the hash.

**Comment by globin** [comment]
Changed this to use `fetchFromGitHub`, thanks :)

**Pull request merged**